composeApp/src/commonMain/kotlin/com/sangyoon/parkingpass/presentation/ui/SessionListScreen.kt

```
@@ -421,11 +418,49 @@ private fun formatDateTime(dateTime: String): String {
+            diffHours -= 1
+        }
+
+        if (diffHours < 0) diffHours += 24
+
+        when {
+            diffHours > 0 && diffMinutes > 0 -> "${diffHours}시간 ${diffMinutes}분"
+            diffHours > 0 -> "${diffHours}시간"
+            diffMinutes > 0 -> "${diffMinutes}분"
+            else -> "1분 미만"
+        }
+    } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)